doc/dev/mgmt/tests.md

```diff
@@ -15,178 +14,136 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
-and rename it `mgmt_settings_real.py`.
+In live mode, you need to use real credentials like those you obtained in the previous section. To enable the tests to use them, make a copy of the `mgmt_settings_fake.py` file in the same location, and rename it `mgmt_settings_real.py`.
 Then make the following changes:
 
-*   Change the value of the `SUBSCRIPTION_ID` constant to your subscription ID.
-    (If you don't have it,
-    you can find it in the "Overview" section of the "Subscriptions" blade
-    in the [Azure portal](https://portal.azure.com/).)
-*   Change the `get_credentials()` function to construct and return
-    a `UserPassCredentials` or `ServicePrincipalCredentials` object
-    such as you constructed in the samples in the previous section.
-    (Don't forget to make sure the necessary imports are present as well!)
+* Change the value of the `SUBSCRIPTION_ID` constant to your subscription ID. (If you don't have it, you can find it in the "Overview" section of the "Subscriptions" blade in the [Azure portal](https://portal.azure.com/).)
+* Change the `get_credentials()` function to construct and return a `UserPassCredentials` or `ServicePrincipalCredentials` object such as you constructed in the samples in the previous section. (Don't forget to make sure the necessary imports are present as well!)
+```python
+def get_credentials(**kwargs):
+    import os
+    from azure.common.credentials import ServicePrincipalCredentials
+
+    return ServicePrincipalCredentials(
+        client_id = os.environ['AZURE_CLIENT_ID'],
+        secret = os.environ['AZURE_CLIENT_SECRET'],
+        tenant = os.environ['AZURE_TENANT_ID']
+    )
+```
+* Change the `get_azure_core_credentials()` function to construct and return a `ClientSecretCredential:
+```python
+def get_azure_core_credentials(**kwargs):
+    from azure.identity import ClientSecretCredential
+    return ClientSecretCredential(
+        client_id = os.environ['AZURE_CLIENT_ID'],
+        client_secret = os.environ['AZURE_CLIENT_SECRET'],
+        tenant_id = os.environ['AZURE_TENANT_ID']
+    )
+```
 
 **Important: `mgmt_settings_real.py` should not be committed since it contains your actual credentials! To prevent this, it is included in `.gitignore`.**
 
 ## Running tests in live mode
 
 To configure the tests to run in live mode, you have two options:
 
-*   Set the environment variable `AZURE_TEST_RUN_LIVE` to "true" or "yes".
-    If you want to go back to playback mode you can either unset it entirely
-    or set it to "false" or "no".
-*   Create a `testsettings_local.cfg` file in the same directory as
-    `mgmt_settings_real.py`. It should look like the following:
+* Set the environment variable `AZURE_TEST_RUN_LIVE` to "true" or "yes". If you want to go back to playback mode you can either unset it entirely or set it to "false" or "no".
+* Create a `testsettings_local.cfg` file in the same directory as `mgmt_settings_real.py`. It should look like the following:
```

Comment:
Let's provide the full folder path instead of saying "search for mgmt_settings_real.py and that;'s the folder". The less I talk about mgmt_settings_real.py the better I feel :p